### PR TITLE
CodeStyle for Authenticationplugin: ladp

### PR DESCRIPTION
**Pull request by zero-24**
This PR fixes some CodeStyle for Authenticationplugin: ladp

### CODE REVIEW

File: plugins/authentication/ldap/ldap.php [hunk 1]
```diff
@@ -94,18 +94,19 @@ public function onUserAuthenticate($credentials, $options, &$response)
...
 						$response->error_message = JText::_('JGLOBAL_AUTH_USER_NOT_FOUND');
 					}
 				}
 				else
 				{
-					$response->status = JAuthentication::STATUS_FAILURE;
+					$response->status        = JAuthentication::STATUS_FAILURE;
 					$response->error_message = JText::_('JGLOBAL_AUTH_NO_BIND');
 				}
-			}	break;
```
Comment (zero-24) [inline plugins/authentication/ldap/ldap.php]: is this code correct? It don't looks ok here...

File: plugins/authentication/ldap/ldap.php [hunk 2]
```diff
@@ -116,10 +117,11 @@ public function onUserAuthenticate($credentials, $options, &$response)
 				}
 				else
 				{
-					$response->status = JAuthentication::STATUS_FAILURE;
+					$response->status        = JAuthentication::STATUS_FAILURE;
 					$response->error_message = JText::_('JGLOBAL_AUTH_BIND_FAILED');
 				}
-			}	break;
```
Comment (zero-24) [inline plugins/authentication/ldap/ldap.php]: same here: is this code correct? It don't looks ok here...

File: plugins/authentication/ldap/ldap.php [hunk 3]
```diff
@@ -30,10 +30,10 @@ class PlgAuthenticationLdap extends JPlugin
 	public function onUserAuthenticate($credentials, $options, &$response)
 	{
 		$userdetails = null;
-		$success = 0;
+		$success     = 0;
 		$userdetails = array();
 
-		// For JLog
+		// Se the response type for JLog
```
Comment (losedk) [inline plugins/authentication/ldap/ldap.php]: See*

Comment (zero-24) [inline plugins/authentication/ldap/ldap.php]: thanks fixed but i mean `Set the response type for JLog` :smile:

File: plugins/authentication/ldap/ldap.php [hunk 4]
```diff
@@ -94,18 +94,19 @@ public function onUserAuthenticate($credentials, $options, &$response)
...
 				}
 				else
 				{
-					$response->status = JAuthentication::STATUS_FAILURE;
+					$response->status        = JAuthentication::STATUS_FAILURE;
 					$response->error_message = JText::_('JGLOBAL_AUTH_NO_BIND');
 				}
-			}	break;
+			}
+			break;
```
Comment (javigomez) [inline plugins/authentication/ldap/ldap.php]: I haven't checked against the sniffer (https://github.com/joomla/coding-standards#joomla-coding-standards) but I'm almost sure that there should be a blank line after closing bracket. It should be:

```
}

break;
```

Comment (zero-24) [inline plugins/authentication/ldap/ldap.php]: thanks fixed but i think the place that the `break;` is located is wrong in case of funtionalitiy / syntax

**Comment by zero-24** [comment]
Closing as we have no response here. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5762">issues.joomla.org/joomla-cms/5762</a>.</sub>

**Pull request closed without merging**